hack/testdata/testpilot.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: navigator.jetstack.io/v1alpha1
+kind: Pilot
+metadata:
+  name: bar
+  namespace: foo
+  ownerReferences:
+  - apiVersion: navigator.jetstack.io/v1alpha1
+    kind: ElasticsearchCluster
+    name: escluster
+    uid: 23c88696-cf7b-11e7-9ec9-0a580a200927
```

Comment:
Not sure we should be hardcoding a uid here - really, navigator-controller should be responsible for creating this resource even in a test environment. Otherwise we could end out seeing test failures because this fixture file hasn't been updated to the newest format (e.g. if we add some new field/remove something/rename something)